src/model/actions-metadata-model.ts

```diff
@@ -91,6 +91,22 @@ export interface Input {
    * @default - none
    */
   readonly deprecationMessage?: string;
+
+  /**
+   * PROJEN ONLY: Specify the type of input. This will be used to
+   * generate sample code, and is not part of GitHub Input Syntax.
+   *
+   * @default Type.STRING
+   */
+  readonly type?: Type;
```

Comment:
If this is projen only then shouldn't it be separated from the model? I feel like this ought to be refactored into two interfaces, e.g. `InputCommon`/`InputAttributes` and `Input`